src/components/dashboard/Dashboard.js

```diff
@@ -103,7 +103,7 @@ export class Dashboard extends Component {
     const { database, t } = this.props;
     const { spaceId } = this.state;
 
-    if (!database || _.isEmpty(database)) {
+    if (database) {
```

Comment:
Can you explain how this works?

Reply:
Since we are already checking the database condition (empty and undefined) in the main `render` function, we can remove the empty check in the render of the filters. (We might remove the whole condition in the filters though). However I see there is a typo here with the missing `!`